.github/workflows/release.yml

```diff
@@ -19,12 +19,24 @@ jobs:
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - name: Flatten charts directory
+        env:
+          CHARTS: charts/consul charts/consul-backup-gcs/consul-backup-gcs charts/gcloud-cron/gcloud-cron charts/vault/incubator/vault charts/basisai/consul-esm charts/basisai/telegraf charts/fluent-bit/stable/fluent-bit charts/kube-janitor charts/bulldozer-deployment/bulldozer-bot charts/eks-container-insights/eks-container-insights charts/gcp-billing-slack-notify/gcp-billing-slack-notify charts/cert-manager-aio/cert-manager-aio charts/basisai/default-psp charts/basisai/cloudsql-proxy
+          CHARTS_DIR: flatten_charts
+        run: |
+          git checkout -b flatten-charts
+          mkdir $CHARTS_DIR
+          cp -r $CHARTS $CHARTS_DIR
+          ls -al $CHARTS_DIR
+          git add .
+          git commit -m "flatten-charts"
```

Comment:
Got to force it because https://github.com/helm/chart-releaser-action/blob/8505b7acc8f413fdf6e6d38b4f3b98282eea7833/cr.sh#L200